examples_tests/41.VisibilityBuffer/main.cpp

```diff
@@ -1191,6 +1192,23 @@ int main()
         }
     };
 
+    using DPG = ext::DepthPyramidGenerator::DepthPyramidGenerator;
+
+    DPG dpg(driver, am, depthBufferView);
+
+    const uint32_t mipCnt = DPG::getMaxMipCntFromImage(depthBufferView);
+    core::vector<core::smart_refctd_ptr<IGPUImageView>> mips(mipCnt);
+    DPG::createMipMapImageViews(driver, depthBufferView, mips.data());
+
+    core::smart_refctd_ptr<IGPUDescriptorSetLayout> dpgDsLayout;
+    core::smart_refctd_ptr<IGPUDescriptorSet> dpgDs;
+    auto dsLayoutDsTmp = DPG::createDescriptorSet(driver, depthBufferView, mips.data());
```

Comment:
if you're going to generate the depth pyramid in multiple dispatches, you also need multiple descriptor sets (each dispatch takes a different set)